packages/Webkul/Admin/src/Http/Controllers/Settings/DataTransfer/ImportController.php

```diff
@@ -126,32 +137,21 @@ public function edit(int $id)
      */
     public function update(int $id)
     {
-        $importers = array_keys(config('importers'));
+        $importerConfig = config('importers');
+
+        $importers = array_keys($importerConfig);
 
         $import = $this->jobInstancesRepository->findOrFail($id);
 
         $this->validate(request(), [
             'code'                => 'required',
             'entity_type'         => 'required|in:'.implode(',', $importers),
-            'action'              => 'required:in:append,delete',
-            'validation_strategy' => 'required:in:stop-on-errors,skip-errors',
-            'allowed_errors'      => 'required|integer|min:0',
-            'field_separator'     => ['required', new SeparatorTypes],
-            'file'                => 'mimes:csv,xls,xlsx,txt',
         ], ['file.mimes' => trans('core::validation.file-type')]);
 
         Event::dispatch('data_transfer.imports.update.before');
 
         $data = array_merge(
-            request()->only([
-                'entity_type',
-                'action',
-                'validation_strategy',
-                'validation_strategy',
-                'allowed_errors',
-                'field_separator',
-                'images_directory_path',
-            ]),
+            request()->except(['_token', 'code']),
```

Comment:
Use only here to limit what data is allowed also just add 'filters' for its value